.github/workflows/sonar.yaml

```diff
@@ -26,7 +26,7 @@ jobs:
 
       - run: pytest --cov=yoti_python_sdk yoti_python_sdk/tests --cov-report=xml:coverage-reports/coverage-new.xml
 
-      - run: sed -i 's+<source>.*</source>+<source>/home/travis/build/getyoti/yoti-python-sdk/yoti_python_sdk</source>+g' coverage-reports/coverage-new.xml
+      - run: sed -i 's+<source>.*</source>+<source>/github/workspace</source>+g' coverage-reports/coverage-new.xml
```

Comment:
Doesn't look like this is working, but we can address in a separate PR to avoid inflating this one